persistence/src/main/java/com/epam/rft/atsy/persistence/repositories/ApplicationsRepository.java

```diff
@@ -19,13 +17,4 @@
    * @return the list of ApplicationEntities
    */
   List<ApplicationEntity> findByCandidateEntity(CandidateEntity candidateEntity);
-
-  /**
-   * Returns a page of ApplicationEntities of the candidate.
-   * @param candidateEntity the given candidate
-   * @param pageable a pageable object
-   * @return a page of ApplicationEntities
-   */
-  Page<ApplicationEntity> findByCandidateEntity(CandidateEntity candidateEntity, Pageable pageable);
```

Comment:
Merge of the master branch was not successful.
Features committed into master branch are removed in this branch and accepting this pull request would destroy those functionalities.